site/slices/contact-us-slice/form/index.js

```diff
@@ -31,42 +31,47 @@ class Form extends Component {
     return (
       <section className={styles.formContainer} id="contactUs">
         <h2 className={styles.heading}>
-          We don’t have a sales team.
+          Project in mind?
           <br />
-          Speak to one of our founders.
+          Tell us more.
         </h2>
 
         <form className={styles.contactUsForm}>
-          <label className={styles.formLabel} htmlFor="message">
-            Message:&nbsp;
-            <span className={styles.errorMessage}>{errors.message}</span>
-          </label>
-          <textarea
-            rows="5"
-            className={cx({
-              inputBox: true,
-              hasErrors: errors.message,
-            })}
-            name="message"
-            id="contactUsMessage"
-            defaultValue={message}
-            onChange={this.handleInputChange}
-          />
+          <div className={styles.inputContainer}>
+            <label className={styles.formLabel} htmlFor="message">
+              Message:&nbsp;
+              <span className={styles.errorMessage}>{errors.message}</span>
+            </label>
+            <textarea
+              rows="5"
+              className={cx({
+                inputBox: true,
+                hasErrors: errors.message,
+              })}
+              name="message"
+              id="contactUsMessage"
+              defaultValue={message}
+              onChange={this.handleInputChange}
+            />
+          </div>
 
-          <label className={styles.formLabel} htmlFor="contact">
-            Your email:&nbsp;
-            <span className={styles.errorMessage}>{errors.contact}</span>
-          </label>
-          <input
-            className={cx({
-              inputBox: true,
-              hasErrors: errors.contact,
-            })}
-            name="contact"
-            type="text"
-            value={contact}
-            onChange={this.handleInputChange}
-          />
+          <div className={styles.inputContainer}>
+            <label className={styles.formLabel} htmlFor="contact">
+              Your email:&nbsp;
+              <span className={styles.errorMessage}>{errors.contact}</span>
+            </label>
+            <input
+              className={cx({
+                inputBox: true,
+                hasErrors: errors.contact,
+              })}
+              name="contact"
+              type="text"
+              onChange={this.handleInputChange}
+              value={contact}
+              placeholder="sam@company.com"
```

Comment:
Maybe change this to `hello@red-badger.com` ? The problem with the above email is that it might actually be someone's email.